torch/nn/utils/clip_grad.py

```diff
@@ -25,9 +25,12 @@ def clip_grad_norm_(parameters, max_norm, norm_type=2):
     max_norm = float(max_norm)
     norm_type = float(norm_type)
     if norm_type == inf:
-        total_norm = max(p.grad.detach().abs().max() for p in parameters)
+        total_norm = max(p.grad.detach().abs().max().item() for p in parameters) if len(parameters) != 0 else 0
```

Comment:
Do you want this to be `0.` in the else to make it a float?

Reply:
Yes, I'll make it a float. XLA is making it None in it's patch, I'm not sure if 0. or None makes more sense, but I'll stick with 0 for now.